README.adoc

```diff
@@ -219,30 +328,55 @@ SystemClient.java
+A benefit is that they can be reused across many different containers,
+even if they all require different environment variables to be set with the same value.
 
 Create a ConfigMap to configure the app name with the following `kubectl` command.
 [role='command']
 ```
 kubectl create configmap sys-app-root --from-literal contextRoot=/dev
 ```
 
-This command deploys a ConfigMap named `sys-app-root` to your cluster. It has a key called `contextRoot` with a value of `/dev`. The `--from-literal` flag allows you to specify individual key-value pairs to store in this ConfigMap. Other available options, such as `--from-file` and `--from-env-file`, provide more versatility as to what you want to configure. Details about these options can be found in the https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-configmap-em-[{kube} CLI documentation^].
+This command deploys a ConfigMap named `sys-app-root` to your cluster.
+It has a key called `contextRoot` with a value of `/dev`.
+The `--from-literal` flag allows you to specify individual key-value pairs to store in this ConfigMap.
+Other available options, such as `--from-file` and `--from-env-file`,
+provide more versatility as to what you want to configure.
+Details about these options can be found in the
+https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-configmap-em-[{kube} CLI documentation^].
 
-Create a Secret to configure the new credentials that `inventory` will use to authenticate against `system` with the following `kubectl` command.
+Create a Secret to configure the new credentials that `inventory` will use to
+authenticate against `system` with the following `kubectl` command.
 [role='command']
 ```
 kubectl create secret generic sys-app-credentials --from-literal username=alice --from-literal password=wonderland
 ```
  
-This command looks similar to the command to create a ConfigMap, but one difference is the word `generic`. This word creates a Secret that doesn't store information in any specialized way. There are different types of secrets, such as secrets to store Docker credentials and secrets to store public and private key pairs.
-
-A Secret is similar to a ConfigMap. A key difference is that a Secret is used for confidential information such as credentials. One of the main differences is that you must explicitly tell `kubectl` to show you the contents of a Secret. Additionally, when it does show you the information, it only shows you a Base64 encoded version so that a casual onlooker doesn't accidentally see any sensitive data. Secrets don't provide any encryption by default, that is something you'll either need to do yourself or find an alternate option to configure.
+This command looks similar to the command to create a ConfigMap, but one difference is the word `generic`.
+This word creates a Secret that doesn't store information in any specialized way.
+There are different types of secrets, such as secrets to store Docker credentials
```

Comment:
There are different types of secrets
---->
Different types of secrets are avaialble